#! /bin/sh -e
## 20_conffile.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use a default and template conffile

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad smokeping-1.37/etc/config.dist /tmp/dpep.kEWhOr/smokeping-1.37/etc/config.dist
--- smokeping-1.37/etc/config.dist	2005-01-29 23:14:21.000000000 +0100
+++ /tmp/dpep.kEWhOr/smokeping-1.37/etc/config.dist	2005-01-29 23:14:22.000000000 +0100
@@ -1,19 +1,29 @@
 # Note that all IP addresses in this file are false, to prevent some
-# machine falling uder a deadly DOS storm because all users keep
+# machine falling under a deadly DOS storm because all users keep
 # the same addresses in their config.
 
 *** General ***
 
+################################################
+# DON'T TOUCH UNLESS YOU KNOW WHAT YOU'RE DOING
+# BETWEEN THESE MARKS!
+################################################
+sendmail = /usr/lib/sendmail
+imgcache = /var/www/smokeping
+imgurl   = ../smokeping
+datadir  = /var/lib/smokeping
+piddir   = /var/run/smokeping
+smokemail = /etc/smokeping/smokemail
+################################################
+# END OF DON'T TOUCH SECTION
+################################################
+
+
+# Please edit this to suit your installation
 owner    = Joe Random
 contact  = joe@some.place.xyz
+cgiurl   = http://put_here_your_domain/cgi-bin/smokeping.cgi
 mailhost = smtp.mailhost.abc
-sendmail = /usr/lib/sendmail
-imgcache = /home/oetiker/public_html/.simg
-imgurl   = ../.simg
-datadir  = /home/oetiker/data/projects/AADJ-smokeping/dist/var
-piddir  = /home/oetiker/data/projects/AADJ-smokeping/dist/var
-cgiurl   = http://people.ee.ethz.ch/~oetiker/smokeping/smokeping.cgi
-smokemail = /home/oetiker/data/projects/AADJ-smokeping/dist/etc/smokemail.dist
 # specify this to get syslog logging
 syslogfacility = local0
 # each probe is now run in its own process
@@ -65,7 +75,7 @@
 
 *** Presentation ***
 
-template = /home/oetiker/data/projects/AADJ-smokeping/dist/etc/basepage.html.dist
+template = /etc/smokeping/basepage.html
 
 + overview 
 
@@ -88,16 +98,20 @@
 
 + FPing
 
-binary = /usr/sepp/bin/fping
+binary = /usr/bin/fping
 
 *** Targets ***
 
 probe = FPing
 
+## You have to edit and uncomment all what you want below this.
+# Please, refer to smokeping_config man page for more info
+# The given adresses aren't real to avoid DoS.
+
 menu = Top
 title = Network Latency Grapher
-remark = Welcome to the SmokePing website of xxx Company. \
-         Here you will learn all about the latency of our network.
+remark = Welcome to the SmokePing website of 'A poorly \
+	mantained site running Debian.'
 
 + World
 
@@ -115,62 +129,62 @@
 title =Swiss Connectivity
 alerts = bigloss,someloss,startloss
 
-++++ SBB
-
-menu = SBB/CFF/FFS
-title =Swiss Federal Railways Webserver
-host = www.railway-server.abc
-
+# ++++ SBB
+#
+# menu = SBB/CFF/FFS
+# title =Swiss Federal Railways Webserver
+# host = www.railway-server.abc
 
-++++ Tiscali
 
-menu = Tiscali Web
-title = Tiscali Webserver www.tiscali.abc
-host = www.tiscali-web.abc
+# ++++ Tiscali
+#
+# menu = Tiscali Web
+# title = Tiscali Webserver www.tiscali.abc
+# host = www.tiscali-web.abc
 
 +++ UK
 
 menu = United Kingdom
 title = United Kingdom
 
-++++ UCL
-
-menu = UCL
-title = UCL
-host = www.ucl-abc.acc.uk
+# ++++ UCL
+#
+# menu = UCL
+# title = UCL
+# host = www.ucl-abc.acc.uk
 
 ++ USA
 
 menu = North America
 title =North American Connectivity
 
-+++ MIT
-
-menu = MIT
-title = Massachusetts Institute of Technology Webserver
-host = www.gurkoman.ybc
-
-+++ IU
-
-menu = IU
-title = Indiana University
-host = www.iu.ali
-
-+++ UCB
+# +++ MIT
 
-menu = U. C. Berkeley
-title = U. C. Berkeley Webserver
-host = www.berkly.udi
+# menu = MIT
+# title = Massachusetts Institute of Technology Webserver
+# host = www.gurkoman.ybc
 
-+++ UCSD
+# +++ IU
+# 
+# menu = IU
+# title = Indiana University
+# host = www.iu.ali
 
-menu = U. C. San Diego
-title = U. C. San Diego Webserver
-host = www.ucsdddar.art
+# +++ UCB
+#
+# menu = U. C. Berkeley
+# title = U. C. Berkeley Webserver
+# host = www.berkly.udi
 
-+++ Sun
+# +++ UCSD
+#
+# menu = U. C. San Diego
+# title = U. C. San Diego Webserver
+# host = www.ucsdddar.art
 
-menu = Sun Microsystems
-title = Sun Microsystems Webserver
-host = www.sun-web.com
+# +++ Sun
+#
+# menu = Sun Microsystems
+# title = Sun Microsystems Webserver
+# host = www.sun-web.com
 
